packages/updating-element/src/updating-element.ts

```diff
@@ -850,6 +862,10 @@ export abstract class UpdatingElement extends HTMLElement {
       // Ensure any previous update has resolved before updating.
       // This `await` also ensures that property changes are batched.
       await this._updatePromise;
+      // If we were disconnected, wait until re-connected to flush an update
+      while (this._pendingConnectionPromise) {
```

Comment:
Is there good test coverage for this while needing to be there?

Reply:
Added here: https://github.com/Polymer/lit-html/pull/1432/files/a0c5122f469acf219e0125549514431c1483d603#diff-afa936faa2534e921592fa2c0672a14d38d5b9963b8e2c752e46fcb6e700effaR2847